src/api/urls.py

```diff
@@ -21,7 +27,11 @@
         url(r'offers/$', offers),
         url(r'(?P<lan_code>[-\w]+)/sent_push_notifications/$', sent_push_notifications),
         url(r'(?P<languages>[-\w]+)/feedback/$', page_feedback.feedback),
+        url(r'(?P<language_code>[-\w]+)/feedback/cities$', region_list_feedback.region_list_feedback),
```

Comment:
I think the url to the region list feedback was incorrect in the APIv3 docu. It should not contain the region slug and language code I think...